### PR TITLE
customized side by side

### PR DESCRIPTION
**Pull request by WenjiaoYue**
## Type of Change

feature 

## Description

add customized side by side project into ITREX
JIRA ticket: (https://jira.devtools.intel.com/browse/NLPTOOLKIU-1253)

## Expected Behavior & Potential Risk

merged into ITREX

## How has this PR been tested?

Refer to read me, after having node and npm environment, npm install and npm run dev

## Dependency Change?

no

### CODE REVIEW

File: intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CHAT_A100_URL = 'http://10.112.228.151:80/v2/rag/chat'
```
Comment (hshen14) [inline intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/.env:1]: shall we use dummy url?

Comment (WenjiaoYue) [reply intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/.env:1]: Yeah! it has been removed!

**Review by hshen14 — APPROVED** [review]

File: intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+![project-screenshot](https://imgur.com/6mf33rM.png)
+![project-screenshot](https://imgur.com/DJCXwNL.png)
+![project-screenshot](https://imgur.com/1XhrYdA.png)
+
+
+<h2>🧐 Features</h2>
+
+Here're some of the project's features:
+
+- Start a Talking Chat：Compare the performance in output quality and response time between Gaudi2 and A100 using textual conversations.
```
Comment (lvliang-intel) [inline intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/README.md:15]: not talking chat?

File: intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+3. Modify the required .env variables.
+    ```
+    CHAT_A100_URL = ''
+    CHAT_GAUDI2_URL = ''
+    KNOWLEDGE_A100_URL = ''
+    KNOWLEDGE_GAUDI2_URL = ''
+    ```
+4. Execute `npm install` to install the corresponding dependencies.
+
+5. Execute `npm run dev` in both environments
```
Comment (lvliang-intel) [inline intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/README.md:34]: Should add one part about how to run it on Huggingface Space.

File: intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/README.md (hunk 2 above)
Comment (WenjiaoYue) [reply intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/README.md]: It has been modified.

File: intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/README.md (hunk 3 above)
Comment (WenjiaoYue) [reply intel_extension_for_transformers/neural_chat/ui/customized/side_by_side/README.md:34]: No need to add in this repo.

**Review by lvliang-intel — APPROVED** [review]

**Review by hshen14 — APPROVED** [review]

**Pull request merged**